### PR TITLE
bump TaylorIntegration.jl

### LINKED ISSUE #138: Update to TaylorIntegration v0.10
CompatHelper fell asleep again...

### PR DESCRIPTION
**Pull request by lucaferranti**
fixes #138

### CODE REVIEW

**Comment by schillic** [comment]
Thanks, can you also tag a new release?

**Comment by lucaferranti** [comment]
>  Thanks, can you also tag a new release?

after the PR is merged yes

**Review by lucaferranti — COMMENTED** [review]
bump patch version

File: Project.toml [hunk 1]
```diff
@@ -19,7 +19,7 @@ IntervalArithmetic = "^0.20"
 IntervalRootFinding = "0.5"
 RecipesBase = "1"
 Reexport = "1"
-TaylorIntegration = "0.9"
+TaylorIntegration = "0.9, 0.10"
 TaylorSeries = "0.12"
 julia = "1.6, 1.7"
```
Comment (lbenet) [inline Project.toml:24]: Thanks @lucaferranti for solving this. What about also increasing the julia version to include 1.8?

**Review by lbenet — APPROVED** [review]
I left one very minor comment, that is perhaps not needed. Aside from that, LGTM, so please go ahead and merge the changes.

File: Project.toml (hunk 1 above)
Comment (schillic) [reply Project.toml:24]: AFAIK, once the major version is bigger than 0, you only have to list the supported major versions. So the 1.7 is already redundant. The next version you would have to add is 2.

Comment (lucaferranti) [reply Project.toml:24]: Yep, exactly

Comment (lucaferranti) [reply Project.toml:24]: Having `julia = "1.6"` means [1.6, 2.0)

Comment (lbenet) [reply Project.toml:24]: Thanks @schillic for the clarification! So, let's go ahead and merge this and register the new version!

**Comment by lbenet** [comment]
@lucaferranti Can you take care of registering the new version?

**Comment by lucaferranti** [comment]
Yes

**Pull request merged**